src/components/DataTable/Cells/contracts.js

```diff
@@ -19,18 +25,26 @@ const endDate = {
     }
     return endDate.format('Do MMMM YYYY');
   },
+  sortMethod: (a, b) => (moment(a).isBefore(moment(b)) ? 1 : -1),
```

Comment:
could this function be assigned to a variable - duplication on line 10 and line 28